### PR TITLE
XW-1741 | add sitename and vertical to footer

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1741

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Comment by hakubo** [comment]
lgtm

**Comment by kvas-damian** [comment]
LGTM

File: server/app/views/_partials/global-footer/global-footer.hbs [hunk 1]
```diff
@@ -30,6 +30,8 @@
 	<div class="wds-global-footer__licensing-and-vertical">
 		<span>{{{i18n
 			model.licensing_and_vertical.description.key
+			sitename=model.licensing_and_vertical.description.params.sitename.value
+			vertical=(i18n model.licensing_and_vertical.description.params.vertical.key ns='design-system')
```
Comment (rogatty) [inline server/app/views/_partials/global-footer/global-footer.hbs]: It's fine for now but I'd like us to think about implementing something like https://github.com/Wikia/app/blob/cde83a738dcad57c287a6687701964e515a58ed1/extensions/wikia/DesignSystem/DesignSystemHelper.class.php#L71-L93 to make it future proof.

**Comment by rogatty** [comment]
LGTM

**Pull request merged**